### PR TITLE
Added Cell Results Widget

### PR DESCRIPTION
**Pull request by hapney**
Unsure if I should remove the dummy data and/or initialize the data somewhere else.

### CODE REVIEW

File: src/batlabmainwindow.cpp [hunk 1]
```diff
@@ -477,8 +492,69 @@ void BatlabMainWindow::showNewCellPlaylistWizard() {
     wizard->show();
 }
 
+cellResultsDisplayInfo createInitializedcellResultsDisplayInfo() {
```
Comment (stephenrparsons) [inline src/batlabmainwindow.cpp]: Thoughts on moving this into Batlablib as it is the sort of thing we could re-use in a command line or other application?

File: src/components/resultswidget.cpp [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+ResultsWidget::ResultsWidget(QVector<cellResultsDisplayInfo> infos, QFrame *parent) : QFrame(parent)
+{
+
+    this->infos = infos;
+
+    // Total ResultsWidget Layout
+    resultsLayout = new QVBoxLayout;
+
+    // Layout for Buttons Only
+    resultsButtonLayout = new QGridLayout;
```
Comment (stephenrparsons) [inline src/components/resultswidget.cpp:12]: Eventually I think we can make these buttons and their layout look a little better, if you have any ideas now go for it (I am no designer but they seem kinda big and spread out)

**Review by stephenrparsons — COMMENTED** [review]
Looks good, left a couple of comments. Take a look and then feel free to merge.

File: src/batlabmainwindow.cpp (hunk 1 above)
Comment (hapney) [reply src/batlabmainwindow.cpp]: Good call! I moved it.

File: src/components/resultswidget.cpp (hunk 2 above)
Comment (hapney) [reply src/components/resultswidget.cpp:12]: I made it a bit better. They are all smaller and only take up the right half of the row.

**Pull request merged**